README.md

```diff
@@ -109,7 +136,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+[myAttemptAtStateMachine.pdf](myAttemptAtStateMachine.pdf)
```

Comment:
Almost there. The only problem is that empty transition from S1 to S2. 

In a state machine you have states (the circles) and directed transitions between the circles. The transitions are each labelled with an event. When you are in a particular state and you receive an event, you look for the transition with that even as a label and follow it to the next state.

Every transition must be caused by an event. Otherwise,  how can the state machine decide when to take it?

Here's your machine but with S1 and S2 combined:

![image](https://user-images.githubusercontent.com/10648/78724595-6aa7f680-78f3-11ea-9c04-9cee36bf1e19.png)

# -2